fern/ir-types/definition/errors.yml

```diff
@@ -21,6 +24,10 @@ types:
       typeV3: optional<types.TypeReference>
       http: optional<HttpErrorConfiguration>
       statusCode: integer
+  ErrorDeclarationDiscriminant:
```

Comment:
Can we call this `ErrorDeclarationDiscriminant` to distinguish between discriminant ('errorName') and discriminant value ('PostNotFoundError')